admin/import/plugins/class-abstract-plugin-importer.php

```diff
@@ -221,6 +234,10 @@ protected function meta_key_clone( $old_key, $new_key, $replace_values = [] ) {
 
 		$this->meta_key_clone_replace( $replace_values );
 
+		// phpcs:ignore WordPress.DB.DirectDatabaseQuery.NoCaching, WordPress.DB.DirectDatabaseQuery.DirectQuery
```

Comment:
This needs a reason in the documentation to explain why we're ignoring (search the codebase for examples)